src/sentry/lang/javascript/processor.py

```diff
@@ -210,16 +210,23 @@ def fetch_release_file(filename, release):
             filename_idents.append(ReleaseFile.get_ident(filename_path))
 
         try:
-            releasefile = ReleaseFile.objects.filter(
+            possible_files = ReleaseFile.objects.filter(
```

Comment:
And I'd coerce this whole queryset into a `list()` since we don't want this lazily evaluated.